conf/distro/opentrons.conf

```diff
@@ -4,7 +4,7 @@ DISTRO = "opentrons"
 DISTRO_NAME = "Opentrons (Opentrons OT3 Distro)"
 DISTRO_VERSION = "0.0.1"
 DISTRO_CODENAME = "ot3"
-DISTRO_FEATURES_append = " systemd"
+DISTRO_FEATURES_append = " systemd wifi bluetooth"
```

Comment:
What are we using bluetooth for?

Reply:
Nothing but the device has it so why not